src/components/CommentCard.jsx

```diff
@@ -17,6 +17,7 @@ const CommentCard = ({ comment, deleteCommentBool, setDeleteCommentBool }) => {
   const deleteThisComment = (event) => {
     event.preventDefault();
     setCommentDeleted(true);
+    setErrorDeleting(false);
     deleteComment(comment.comment_id)
       .then((data) => {})
```

Comment:
if you're not using anything in the .then, you can delete it and just have the function call and the .catch